### ISSUE TITLE
registerTTFont does not work if no path specified

### ISSUE THREAD

**Issue by uli-mierendorff** [zopefoundation/z3c.rml#46]
In the docinit section of an RML file, I usually declare the fonts in the following way
`<registerTTFont faceName="Arial" fileName="Arial.ttf"/>`
Arial.ttf is located somewhere in the /usr/share/fonts/truetype/ directory, and reportlab is able to find it, just by giving the name "Arial.ttf".

With current version of z3c.rml (3.3.0), however, it fails, as it cannot locate "file:////Arial.ttf". After investigating a bit, I modified document.py:

```
--- document.py.old 2016-04-04 11:57:24.000000000 +0200
+++ document.py 2016-09-16 23:31:39.841520461 +0200
@@ -122,10 +122,9 @@
                      u'be previously registered.'),
         required=True)

-    fileName = attr.File(
+    fileName = attr.String(
         title=u'File Name',
         description=u'File path of the of the TrueType font.',
-        doNotOpen=True,
         required=True)

 class RegisterTTFont(directive.RMLDirective):
```

Now it is working again for me.

The way the registerTTFont element is currently handled is clearly wrong, as reportlab has its own methods for finding fonts. z3c.rml should pass the raw value of the XML attribute fileName to reportlab instead of modifying it (except for the case with "[modulename]/path/...")

**Comment by chucklay** [comment]
Ran into this as well. If no path is specified, the default font directories should be checked.

**Comment by strichter** [comment]
Fixed for 4.1.1.
